app/controllers/show.js

```diff
@@ -12,6 +13,25 @@ export default Ember.Controller.extend({
   queryParams: ['seekto'],
   seekto: null,
 
+  embededPdf: Ember.computed('model.show.customFields', 'site.publicSite.fieldDisplays.[]', function() {
+    let pdfDisplays = this.get('site.publicSite.fieldDisplays').sortBy('order').filterBy('widget', 'pdf');
+    for (let i = 0; i < pdfDisplays.length; i++) {
+      let fd = pdfDisplays[i];
+      let fileField = this.get('model.show.customFields').find((field) => {
+        return field.type === 'file' && fd.get('showField') === field.showField;
+      });
+      if (fileField) {
+        let file = this.get('store').peekRecord('web-file', fileField.value);
+        if (/.+\.pdf$/.test(file.get('name'))) {
+          return {
+            url: 'http://localhost:4200/Cablecast/WebFiles/9635/Lance%20-%20Fill%20Gaps%20Feature%20Request.pdf', //Ember.get(file || {}, 'url'),
```

Comment:
does this need to be updated?

Reply:
good call...
need tot take that out... was just for testing.